corehq/apps/userreports/dbaccessors.py

```diff
@@ -0,0 +1,10 @@
+def get_report_configs_by_data_source(domain, data_source_id):
```

Comment:
Yeah, I did envision part of the process of moving over to using django models to be to rewrite each of these functions with a django query, which will probably in many cases include adding an postgres btree ("index") for each couchdb btree ("view"), in addition to writing the query, and making sure the tests pass. Already though, I've been rewriting a number of them to use the same views so others can be removed, and not organizing them by view make that very marginally easier, and that's the main reason for me. Regardless, both conventions are a lot better than what we were doing before! The difference between them is minor and it's easy enough to move between them when/if we feel like one has tangible benefits